debian/postinst

```diff
@@ -12,7 +12,7 @@ case "$1" in
     configure)
 
         # Create custom sudoers file
-        echo "Defaults env_keep += KLOG_FORCE_FLUSH" > $TMP_FILE
+        echo 'Defaults env_keep += "KLOG_FORCE_FLUSH NO_AT_BRIDGE"' > $TMP_FILE
```

Comment:
@alex5simon, it is good to double check; in this case I had to change the
quotes because I need double quotes to end up in the resulting file. I have
checked that the file is generated correctly.

Best,
Alex

On 26 March 2015 at 12:17, Alex Simon <notifications@github.com> wrote:

> In debian/postinst
> <https://github.com/KanoComputing/kano-toolset/pull/120#discussion_r27207499>
> :
>
> > @@ -12,7 +12,7 @@ case "$1" in
> >      configure)
> >
> >          # Create custom sudoers file
> > -        echo "Defaults env_keep += KLOG_FORCE_FLUSH" > $TMP_FILE
> > +        echo 'Defaults env_keep += "KLOG_FORCE_FLUSH NO_AT_BRIDGE"' > $TMP_FILE
>
> @Ealdwulf <https://github.com/Ealdwulf> are these quotes okay? Just to
> double check, cause it's different from what we used before
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KanoComputing/kano-toolset/pull/120/files#r27207499>.
>